waspc/data/Generator/templates/react-app/src/entities/_entity/components/CreateForm.js

```diff
@@ -116,7 +207,20 @@ export default class {= formName =} extends React.Component {
           </div>
           {=/ showSubmitButton =}
         </form>
+      {=# isXRayModeEnabled =}
+        </>
```

Comment:
```suggestion
        </>
```
Also not sure, but I think you can remove this.